test/evergreen.yml

```diff
@@ -4543,6 +4594,84 @@ tasks:
     name: format-stress-test-disagg-switch-5
     tags: ["stress-test-disagg-fail"]
 
+   # FIXME-WT-14566: Enable disagg leader mode on all platforms once failures have been resolved.
+  - <<: *format-stress-test-disagg-leader-palite
```

Comment:
Thank you for adding this! I think we have too many test/format runs altogether. Could we instead reduce down all disagg test/format to 3. Bit out of scope for this ticket, but also including one the PALM ones.

Reply:
Tracking the task in this ticket: https://jira.mongodb.org/browse/WT-15707